sensor_msgs/msg/CompressedImage.msg

```diff
@@ -1,13 +1,16 @@
 # This message contains a compressed image.
 
-std_msgs/Header header # Header timestamp should be acquisition time of image
+std_msgs/Header header       # Header timestamp should be acquisition time of image
                              # Header frame_id should be optical frame of camera
                              # origin of frame should be optical center of cameara
                              # +x should point to the right in the image
                              # +y should point down in the image
                              # +z should point into to plane of the image
 
-string format                # Specifies the format of the data
+uint32 height                # image height, that is, number of rows
+uint32 width                 # image width, that is, number of columns
+
+string pixel_format          # Specifies the format of the data
```

Comment:
This seems like a needless difference.  I understand that this is slightly more descriptive, but I also don't think it is worth the headache for downstream users to change this.  So I'll suggest putting this back to `format`.

Reply:
Note that `encoding` is the current spelling for this in `sensor_msgs/Image`, which is just _wrong_ in almost every case. Everything else (OpenCV, commercial machine vision libraries, etc) refers to this as "pixel format". Unfortunately, "encoding" is even more confusing when dealing with compressed data, so if there is a chance to fix this properly we should.